.gitignore

```diff
@@ -1,3 +1,14 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+*.lock
+*.log
+
 node_modules
 example/**/dist
-package-lock.json
+
+# Mac OS X
+.DS_Store
+._.*
+._*
+
+# Windows
+Thumbs.db
```

Comment:
These should be in your global .gitignore https://stackoverflow.com/a/7335487

Let's remove from here